js/ui/lookup.js

```diff
@@ -156,6 +150,40 @@ const Lookup = DropDownList.inherit({
 
             focusStateEnabled: false,
 
+            dropDownOptions: {
+                showTitle: true,
+
+                width: function() { return $(window).width() * 0.8; },
```

Comment:
Is it OK in SSR?

Reply:
There is no conceptual difference with the previous version, in which `popupWidth` was defined in this way. Therefore, I think so